editor/app/scene/animation-properties.jsx

```diff
@@ -58,15 +59,18 @@ export class AnimationProperties extends WithNodeState {
             />
           </div>
         )}
+
         <div className="speed">
           <label>Speed</label>
-          <input
-            type="number"
-            step=".1"
-            onChange={this.handletAnimationSpeedChange}
+          <Slider
             value={animationState.speed}
+            min={-1}
+            max={3}
+            onChange={this.handleAnimationSpeedChange}
+            step={1}
```

Comment:
this `step` seems to be leftover?

Reply:
That's required by the `input` range element (forgot to mention that detail 😅): https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#step

Reply:
🤔 that makes me think that  if we change the `step` value here, then the `range` fn will be disconnected from this value. We can solve this with two different approaches:

1. Adding back `step` to `range()`, so both the `input` and the `labels` are in sync.
2. Hard-code the `step` value in the `input[range]` component.

I'm slightly inclined for opt. 2, and as I said, if we need to change that later, then we can implement opt. 1.

What do you think?

Reply:
Sorry for the confusion. Yeah I was confused.  i thought that by removing the step in the `range` generation function we were removing the use of `step`.  But it seems like were using steps for two purposes.

1. for creating the range for the labels.
2. controlling the granularity in the input control.

I think what you have is great as is!!  We can remove it if you wanted to since the default value is already `1` in the component but it doesnt matter much at all.

Reply:
Removing `steps` for now, and will include it if we ever need it.

Reply:
actually - i never really asked you.  do you prefer keeping `step` everywhere the way you had it?